zipkin-server/src/main/java/zipkin2/server/internal/ZipkinQueryApiV2.java

```diff
@@ -202,6 +204,12 @@ public AggregatedHttpResponse getAutocompleteValues(
     return maybeCacheNames(values.size() > 3, values, ctx.alloc());
   }
 
+  @Trace("regex:^.*$")
```

Comment:
couldn't find a global toggle to disallow it, hence this workaround. @anuraaga does this please you ?

Reply:
@ikhoon @minwoox ^^

Reply:
How about using route?
```java
in ZipkinHttpConfiguration.java

sb.route()
  .methods(HttpMethod.TRACE)
  .pathPrefix("/")
  .build((ctx, req) -> HttpResponse.of(HttpStatus.FORBIDDEN));
```

Reply:
@jorgheymans Created an issue for that. 😄 
https://github.com/line/armeria/issues/3114

Reply:
I think a decorator would be better to forbid TRACE requests because it could have higher priority HTTP services. 
```java
sb.routeDecorator()
  .methods(HttpMethod.TRACE)
  .pathPrefix("/")
  .build((delegate, ctx, req) -> HttpResponse.of(HttpStatus.FORBIDDEN));              
```

Reply:
thanks for the suggestions, i added the routeDecorator.